java-security/src/main/java/com/sap/cloud/security/json/DefaultJsonObject.java

```diff
@@ -190,4 +190,8 @@ private JSONObject createJsonObject(String jsonString) {
 		}
 	}
 
+	@Override
+	public String toString() {
+		return jsonObject.toString(2);
```

Comment:
what does toString(2)?

Reply:
It pretty prints the json object with indentation level 2